hack/setup_db.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -euo pipefail
+set -o errexit
+
+su - postgres -c "mkdir -p /tmp/postgres/data"
```

Comment:
When I was testing this with you I thought we were trying to avoid this approach because it required root or a password prompt.

What changed?

Reply:
@carbonin the `continuous-integration/jenkins/pr-merge` was failing  https://github.com/openshift/assisted-service/pull/2830#issuecomment-954083198 as Jenkins runs as a root user.